db_master_deploy/includes.sh

```diff
@@ -19,6 +17,23 @@ INFO="${0##*/} - ${USER} - ${comment} - [${SYSLOGPID}] - INFO"
 ERROR="${0##*/} - ${USER} - ${comment} - [${SYSLOGPID}] - ERROR"
 
 COMMAND="${0##*/} $* (pid: $$)"
+PSQL() {
```

Comment:
Nice. Personally I'm not a fan of 'aliases' - for readability I prefer using directly the command with options and parmameters - just a question of flawor.

Reply:
it is not common to use upper case for functions

Reply:
i had to replace the variable with a function / alias for better mocking in the unit tests.
Having a function you can mock the function react to various input parameters

Reply:
https://github.com/geoadmin/deploy/pull/40/files#diff-2d489e8c5a3612452c6995c7a079f725R274